libmscore/fret.cpp

```diff
@@ -59,78 +59,65 @@ FretDiagram::FretDiagram(const FretDiagram& f)
       _frets      = f._frets;
       _fretOffset = f._fretOffset;
       _maxFrets   = f._maxFrets;
-      maxStrings  = f.maxStrings;
       font        = f.font;
-      _barre      = f._barre;
       _userMag    = f._userMag;
+      _dots       = f._dots;
+      _markers    = f._markers;
+      _barres     = f._barres;
+      _fingering  = f._fingering;
```

Comment:
Add
```
      _numPos     = f._numPos;
```
and you will have fixed [#280575: Copy&Paste Fretboard diagram’s number is mirrored](
https://musescore.org/en/node/280575) as well.

Reply:
Thank you! I will add this.